Documentation/DevGuide.rst

```diff
@@ -0,0 +1,45 @@
+Developers Guide
+========================
+
+TODO_DOC: this should probably be split up into the different bits it talks about. Better that each entry has a basic section followed by an advanced section, rather than what we have now: a folder of basic pages followed by a folder of advanced pages. 
```

Comment:
This pages is pretty out-of-date, even compared to the rest of the documentation from the wiki.  The TODO line should note that.

Reply:
addressed in 36a919a  with a comment in the TODO note